client/components/select-vertical/suggestion-search.tsx

```diff
@@ -111,17 +111,14 @@ const SelectVerticalSuggestionSearch: FC< Props > = ( {
 		inputRef.current?.focus();
 	};
 
-	const handleSuggestionsSelect = ( {
-		name,
-		label,
-		value,
-	}: {
+	const handleSuggestionsSelect = ( suggestion: {
 		name: string;
 		label: string;
 		value?: string;
+		has_vertical_images?: boolean;
```

Comment:
I believe we don't really need this prop; it is not needed for conforming to the `Suggestion` interface. Can we remove it? (the `suggestion` arg is not being destructured here, so everything will be passed to `onSelect()`.)

Reply:
Ah yeah you're right 🙂